### PR TITLE
For #211: Redis installation

### PR DESCRIPTION
**Pull request by paulodamaso**
For #211: Redis installation
- Added `Redis` installation instructions to `README.md`
- Added `Redis` installations execution to `docker-compose.yaml`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#229](https://github.com/timelesslounge/timelessis/issues/229) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#229](https://github.com/timelesslounge/timelessis/issues/229) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/b81c4b91-8c04-4668-9a86-57b65eff34aa); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: docker-compose.yaml [hunk 1]
```diff
@@ -40,5 +40,8 @@ services:
       POSTGRES_DB: timelessdb_test
       POSTGRES_USER: timeless_user
       POSTGRES_PASSWORD: timeless_pwd
+  redis:
```
Comment (vryazanov) [inline docker-compose.yaml]: @paulodamaso 
```
  redis:
    image: 'redis:3.2'
    ports:
      - '6379:6379'
```

Comment (vryazanov) [reply docker-compose.yaml]: @paulodamaso also pls add `redis` in `depends_on` section for `app` service:
```yaml
app:
    ....
    depends_on:
      - db
      - redis
```

**Comment by paulodamaso** [comment]
@vryazanov Done, please take a look

File: README.md [hunk 2]
```diff
@@ -227,6 +227,17 @@ psql -U postgres -d timelessdb_dev
...
 
+### Redis Installation
+We use Redis for cache storage. For installing Redis:
+```
+sudo apt install make gcc libc6-dev tcl
+wget http://download.redis.io/redis-stable.tar.gz
+tar xvzf redis-stable.tar.gz
+cd redis-stable
+sudo make install
+src/redis-server > /dev/null &
```
Comment (ammaratef45) [inline README.md]: @paulodamaso  The `/dev/null &` part was added for rultor not to be stuck in the command.
Users would like to see the output and know everything is okay.
please make it `src/redis-server` - the command makes a good draw in the output anyway 😅

File: docker-compose.yaml (hunk 1 above)
Comment (ammaratef45) [reply docker-compose.yaml]: @vryazanov thanks for the help

**Comment by ammaratef45** [comment]
@paulodamaso just added one more comment, please take a look

**Comment by paulodamaso** [comment]
@ammaratef45 Done, please take a look

**Comment by ammaratef45** [comment]
@paulodamaso thanks

**Comment by ammaratef45** [comment]
@rultor merge please

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@emilianodellacasa Didn't even touched this file...

**Comment by emilianodellacasa** [comment]
@paulodamaso not your fault, test dir was excluded before. Please fix it otherwise we cannot merge

**Comment by emilianodellacasa** [comment]
@paulodamaso never mind I will do it manually

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 4 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#229](https://github.com/timelesslounge/timelessis/issues/229) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/timelesslounge/timelessis/issues/229#issuecomment-463093266))

@ypshenychka There is no quality review for [#229](https://github.com/timelesslounge/timelessis/issues/229), no performer

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by ammaratef45** [comment]
@ypshenychka Is this a bug?

> @0crat quality acceptable



> > @0crat quality acceptable ([here](https://github.com/timelesslounge/timelessis/issues/229#issuecomment-463093266))
> 
> @ypshenychka There is no quality review for [#229](https://github.com/timelesslounge/timelessis/issues/229), no performer

@ypshenychka Is this a bug?

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/timelesslounge/timelessis/issues/229#issuecomment-463099346); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Comment by ypshenychka** [comment]
@ammaratef45 Yes. It happens from time to time.

**Pull request merged**